src/Receipt/DonationReceipt.php

```diff
@@ -134,7 +134,7 @@ private function getDonorEmailLineItem() {
 	private function getDonorBillingAddressLineItem() {
 		$address = getDonationDonorAddress( $this->donationId );
 		$address = sprintf(
-			'%1$s<br>%2$s%3$s,%4$s%5$s<br>%6$s',
+			'%1$s<br>%2$s%3$s, %4$s%5$s<br>%6$s',
```

Comment:
So basically, here is the source of the problem. If you just replacing the `<br>` tags with the newline character, the problem will be resolved. The CSS rule which will handle this properly is already there.

Reply:
fixed: https://github.com/impress-org/givewp/compare/959aaa5b55ed...2f9f4a276fad